pom.xml

```diff
@@ -424,7 +424,7 @@
             <dependency>
                 <groupId>org.springframework</groupId>
                 <artifactId>spring-webmvc-portlet</artifactId>
-                <version>${spring.version}</version>
+                <version>4.3.30.RELEASE</version>
```

Comment:
I don't understand this fix.
The value is the same as the variable was before...
https://github.com/uPortal-Project/NewsReaderPortlet/blob/905a0c623e15077873ad06c317782579ae592f22/pom.xml#L83

Reply:
Think I made a mistake here somewhere. Going to close this for now.